hedera-mirror-datagenerator/src/main/java/com/hedera/datagenerator/sdk/supplier/consensus/ConsensusSubmitMessageTransactionSupplier.java

```diff
@@ -55,29 +55,12 @@
     @Getter(lazy = true)
     private final TopicId consensusTopicId = TopicId.fromString(topicId);
 
-    @Getter(lazy = true)
-    private final byte[] messageSuffix = randomByteArray();
-
     @Override
     public TopicMessageSubmitTransaction get() {
         return new TopicMessageSubmitTransaction()
                 .setMaxTransactionFee(Hbar.fromTinybars(maxTransactionFee))
-                .setMessage(generateMessage())
+                .setMessage(!message.isEmpty() ? message.getBytes(StandardCharsets.UTF_8) : Utility
```

Comment:
nit: for readability better to do 
```suggestion
                .setMessage(message.isEmpty() ? Utility.generateMessage(messageSize) : message.getBytes(StandardCharsets.UTF_8)
```